src/test/java/hudson/tasks/AntTest.java

```diff
@@ -123,6 +124,12 @@ private void verify() throws Exception {
         assertEquals(1,l.length);
         r.assertEqualBeans(l[0],new AntInstallation("myAnt","/tmp/foo", JenkinsRule.NO_PROPERTIES),"name,home");
 
+        // Verify that PATH+ANT is set.
```

Comment:
I'll be blunt - I don't think that's necessary. If `buildEnvVars` results in `PATH+ANT` being included, that seems sufficient to me, since we know that `buildEnvVars` gets called and that `PATH+FOO` in `EnvVars` gets added to the `PATH`.